components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -685,6 +692,14 @@
                                         </table>
                                     </td>
                                 </tr>
+                                <tr id="authorizationCodePlain">
+                                    <td class="leftCol-med"><fmt:message key='authorization.code.validity.period'/></td>
+                                    <td><input id="authorizationCodeValidityPeriod" name="authorizationCodeValidityPeriod"
+                                               type="text"
+                                               value="<%=client.getAuthorizationCodeValidityPeriod()%>"/>
```

Comment:
Shall we encode the values here?

Reply:
Is encoding necessary since `client.getAuthorizationCodeValidityPeriod()` returns a server default?

Reply:
But we can give any value to the text input and add the SP rit? If so, from where is that handled?

Reply:
An Exception is thrown if the user inputs something that cannot be parsed into a Long value.